test/e2e/testdrivers/1.16.yaml

```diff
@@ -25,8 +25,8 @@ DriverInfo:
     exec: true
     snapshotDataSource: true
     multipods: true
-    controllerExpansion: true
-    nodeExpansion: true
+    controllerExpansion: false
```

Comment:
Disabled while https://status.digitalocean.com/incidents/qz40dp1qcs1d is still ongoing; I won't merge the PR before undoing this file change and seeing a green run.